cypress/support/generateArticle.js

```diff
@@ -0,0 +1,10 @@
+const generateArticle = () => {
```

Comment:
you already have this function inside config

Reply:
Yea,I know